11ty/definitions/bierasure.md

```diff
@@ -3,6 +3,7 @@ title: Bierasure
 slug: bierasure
 speech: noun
 defined: true
+excerpt: the questioning or denial of the existance or legitimacy of bisexuality
```

Comment:
This originally included a reference to the Bisexuality definition - I wasn't sure whether the link should be translated through.

Reply:
The link can be left out, as the excerpt will mostly be used for social media previews and there no links allowed there anyway :)